src/execution_plan/ops/shared/print_functions.c

```diff
@@ -16,7 +16,10 @@ static inline void _NodeToString(sds *buf, const char *alias, const char *label)
 }
 
 void TraversalToString(const OpBase *op, sds *buf, AlgebraicExpression *ae) {
-	if(!ae) *buf = sdscatprintf(*buf, "%s", op->name);
+	if(!ae) {
+		*buf = sdscatprintf(*buf, "%s", op->name);
+		return;
```

Comment:
why the return here?

Reply:
The rest of this function continues to access `ae`.